views/partials/patient-status-history-dl.html.twig

```diff
@@ -0,0 +1,17 @@
+{% for key, field in orgPatientStatusHistoryData %}
+    <dl class="dl-horizontal">
+        <dt>Status</dt>
+        <dd>{{ field.display_status }}</dd>
+        <dt>User</dt>
+        <dd>{{ field.user_email }}</dd>
+        <dt>Site</dt>
+        <dd>{{ field.site_name }}</dd>
+        <dt>Date</dt>
+        <dd>{{ field.created_ts|date('n/j/Y g:ia', app.userTimezone) }}</dd>
+        <dt>Comments</dt>
+        <dd>{{ field.comments }}</dd>
+    </dl>
+    {% if key < orgPatientStatusHistoryData|length - 1 %}
```

Comment:
You can use `not loop.last` instead of having to look at the key.  [Twig documentation](https://twig.symfony.com/doc/1.x/tags/for.html#the-loop-variable)